### DISCUSSION TITLE
How to toggle Skeleton view of a particular 3d model

### DISCUSSION THREAD

**Discussion by shreyasmulay2305** [google/model-viewer#4075] [General]
hi @elalish @cdata 

I am able to show 3D model into model-viewer quite beautifully.

<img width="595" alt="image" src="https://user-images.githubusercontent.com/123137600/215685936-5ae56456-f9cc-47b2-8789-423199b6fa1f.png">

But I require to show 'skeleton' view of that GLB (see the blue gradient lines inside the model).  Something kind of this.. 
 
 
<img width="539" alt="image" src="https://user-images.githubusercontent.com/123137600/215686074-300cc850-7926-4669-a624-5aef3fc80978.png">

How to achieve this in model-viewer ?

Apologies if this is a repeated question.

**Comment by elalish** [comment]
We don't support this as we only do physically-based rendering. I'd recommend forking us and adding that object and material to our internal `ModelScene`. But that's about all the help I can give.

**Reply by shreyasmulay2305** [reply]
Sure Thanks @elalish :)

Can you please let me know how to access ModelScene. Is it Symbol(scene) ?
